ux/README.md

```diff
@@ -5,3 +5,4 @@ This directory records best practices for researching and implementing user expe
 ## Guides
 
 - [Project research and interviews](/ux/project-research-and-interviews.md)
+- [User Research Report template](https://docs.google.com/document/d/1NX83QAKjkJHE-J2q2CjCORpfGqhPk-XgO2HDW6iScGo/edit) (visible to DataMade employees only)
```

Comment:
This seems like it would be a better fit under the ["Reports" section of the "Project research and interviews" guide](https://github.com/datamade/how-to/blob/master/ux/project-research-and-interviews.md#6-reports). In other guides, we'll sometimes put a link to a template as italicized text underneath the section heading (see e.g. [Project collateral](https://github.com/datamade/how-to/blob/master/project-management/collateral.md#project-plan)). Does that seem doable here?